data/templates/wg21.html

```diff
@@ -62,7 +62,13 @@ <h3 class="subtitle" style="text-align:center">$subtitle$</h3>
 <table style="border:none;float:right">
   <tr>
     <td>Document #:</td>
-    <td>$document$</td>
+    <td>
+      $document$
+$if(number)$
+      <a href="https://wg21.link/P$number$">[Latest]</a>
+      <a href="https://wg21.link/P$number$/status">[Status]</a>
```

Comment:
Sorry to be annoying. Can we keep the links just to the text? This more consistent with how other links are handled by the framework, like `<email>` and `[@P1000]`, etc. They exclude the brackets from the link.
```suggestion
      [<a href="https://wg21.link/P$number$">Latest</a>]
      [<a href="https://wg21.link/P$number$/status">Status</a>]
```

Reply:
Works for me.